src/compositions/usePropStore.ts

```diff
@@ -0,0 +1,12 @@
+import { inject } from 'vue'
+import { PropStore, propStoreKey } from '@/models/PropStore'
+
+export function usePropStore(): PropStore {
+  const store = inject(propStoreKey)
+
+  if (!store) {
+    throw 'PropStore not provided!'
```

Comment:
I think most of our compositions throw a `RouterNotInstalledError` if it cannot find something the router is supposed to provide when it is installed as a plugin.